tile_generator/config_unittest.py

```diff
@@ -638,6 +637,151 @@ def test_multiline_yaml(self):
 If the service does not use the correct GUID, the request will fail with a message about how to upgrade.'
     self.assertEqual(tile_metadata['form_types'][0]['property_inputs'][0]['description'], expected)
 
+  def test_standalone_job_type(self):
+    self.config.update({
+      'packages': [{
+        'name': 'some_errand',
+        'type': 'bosh-release',
+        'path': 'does/it/matter.tgz',
+        'jobs': [{
+          'name': 'my-job',
+          'type': 'standalone'
+        }]
+      }]
+    })
+    self.config.validate()
+    self.config.normalize_jobs()
+    expected = {
+      'some_errand': {
+        'name': 'some_errand'
+      }
+    }
+
+    self.assertEqual(self.config['releases']['some_errand']['jobs'][0]['manifest'], expected)
+
+  def test_deploy_all_job_type_with_external_broker(self):
+    self.config.update({
+      'packages': [{
+        'name': 'some_errand',
+        'type': 'external-broker',
+        'path': 'does/it/matter.tgz',
+        'jobs': [{
```

Comment:
These tests are qorrisome because if a job is not specifically `standalone` the original behavior should remain. It seems that by default the `standalone` is used.

Reply:
Only the first new test is for the `standalone` type. The rest are backfilling tests for the existing job types. Line 688 includes the cf properties in the assertion, maintaining the original behaviour.